### PR TITLE
(DED-85) Migrate to the new mdx library

### PR DESCRIPTION
**Pull request by arturfracala**
# Links

- [Jira](https://livechatinc.atlassian.net/browse/DED-85)

# Description

# Review and release

### CODE REVIEW

**Review by bstopyra — APPROVED** [review]
amazing stuff 🎉

File: scripts/redoc.js [hunk 1]
```diff
@@ -6,35 +6,35 @@ const fs = require("fs");
...
-      fs.mkdirSync(tmpDir);
-    }
-  
-    fs.writeFile(`${tmpDir}/redoc-${name}.html`, `${css} ${html}`, function(err) {
-      if (err) return console.log(err);
-      console.log(`Generated ${tmpDir}/redoc-${name}.html successfully!`);
-    });
+  const response = await redoc.loadAndBundleSpec(
+    `.${
+      process.env.CONTEXT ? "/repo" : "/livechat-public-docs"
```
Comment (jakubsikora) [inline scripts/redoc.js:14]: What is `/repo` ?

**Review by jakubsikora — APPROVED** [review]
Looking hot! Great work.

File: scripts/redoc.js (hunk 1 above)
Comment (arturfracala) [reply scripts/redoc.js:14]: It's Netlify's working directory. Entire build fails, if i delete this.

Comment (arturfracala) [reply scripts/redoc.js:14]: It's Netlify's working directory. Entire build fails, if i delete this.

**Pull request merged**